### ISSUE TITLE
Re-enable all disabled tests in UMD

### ISSUE THREAD

**Issue by broskoTT** [tenstorrent/tt-umd#367]
Some tests were disabled, either due to known missing support or due to being flaky. 
Identify all such tests and re-enable them. Create separate issues to track broken tests.

Specifically, uncomment all commented code inside tests/ folder.

**Comment by nbuncicTT** [comment]
Make this more granular, so @pjanevskiTT will open more smaller issues and once this is done, he will close this issue.

**Comment by pjanevskiTT** [comment]
Broken this out into granular per-test issues tracked under #3050. Closing in favor of the umbrella.
